### ISSUE TITLE
Feature request : set / get the GCP Project of the Script

### ISSUE THREAD

**Issue by sebO-fr** [google/clasp#967]
Hi there,

Do you think it could be possible to implement a clasp command to set the GCP project like it can be done over the UI (cf screen shot) ?

Also, a clasp command to get the current GCP project would be nice with "default" value if the GCP project script is still the default project affected when the script is created.

Would be happy to help but really not sure if I have enough skills  ...

Thanks !

![Screenshot 2023-07-06 14 40 35](https://github.com/google/clasp/assets/80100266/b482aadf-b328-4805-aa88-960a35a9318d)

**Comment by fletort** [comment]
i was wondering same thing that you.... but looking to App Ascript WebAPI... this things seems not supported.... So The first thing is to ask to add this feature on the API itself...

**Comment by sqrrrl** [comment]
Yep, want to add this but it's still blocked by the underlying API.

**Comment by sebO-fr** [comment]
OK ! Thanks for the feedback
